### PR TITLE
[issue-148] Update Flink version to 1.6.0

### PR DESCRIPTION
**Pull request by vijikarthi**
Signed-off-by: Vijay Srinivasaraghavan <vijayaraghavan.srinivasaraghavan@emc.com>

**Changes**
  * Updated flink connector version that was built for Flink 1.6.0

**Reason**
- to address https://github.com/pravega/pravega-samples/issues/148

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -9,13 +9,13 @@
 #
 ### dependencies
 pravegaVersion=0.4.0-50.63aea88-SNAPSHOT
-flinkConnectorVersion=0.4.0-103.7971206-SNAPSHOT
+flinkConnectorVersion=0.4.0-100.d588197-SNAPSHOT
```
Comment (RaulGracia) [inline gradle.properties]: This update is conflicting with the change in PR #147. Let's take into account in which order we merge them.

File: gradle.properties [hunk 2]
```diff
@@ -9,13 +9,13 @@
...
 
 ### Pravega-samples output library
 samplesVersion=0.4.0-SNAPSHOT
 
 ### Flink-connector examples
-flinkVersion=1.4.0
+flinkVersion=1.6.0
```
Comment (RaulGracia) [inline gradle.properties]: Could you please confirm that existing samples working with the Flink 1.6?

Comment (vijikarthi) [reply gradle.properties]: Yes, I have tested flink samples and it looks good.

Comment (RaulGracia) [reply gradle.properties]: Great, in this case I approve the changes.

**Pull request merged**